### ISSUE TITLE
Thermal Images

### ISSUE THREAD

**Issue by diffrxction** [dsc-tiu/open-source-ml-ai#4]
Requirements:
1. Documentation
- Add descriptive documentation on thermal imagery and different types of sensors.
- Use cases of thermal imagery
- Types of thermal data
- Temperature as a quantity adds value to what global scenarios

2. Code
- Reading and displaying thermal image.
- Enhancing thermal images.
- Use in temperature detection via computer vision
- 3 channel thermal to single channel thermal conversion

Follow similar structure as mentioned in other issues. Also add your contributions to the `Thermal Imagery` folder

**Comment by tauquirahmed** [comment]
Assign me the coding part of this thermal imaging problem.

**Comment by diffrxction** [comment]
Sure @tauquirahmed, you can cover any 2 points from the code section given above. 

Please try to provide valid documentation in .md format for your code in same sub-folder and also remember to follow the folder structure given above. All the best! ⭐

**Comment by tauquirahmed** [comment]
I will read a thermal image and enhance the thermal image so that it can be processed for temperature detection.

**Comment by Ripan-Roy** [comment]
I can do the rest and do the documentation part.
Assign me.

**Comment by Ripan-Roy** [comment]
Also can we do it in a .ipynb file so that the documentation and code can be done in a single file. 
Also I will provide the overall documentation in the markdown file.

**Comment by diffrxction** [comment]
> I will read a thermal image and enhance the thermal image so that it can be processed for temperature detection.

Sure, go ahead.

**Comment by diffrxction** [comment]
> I can do the rest and do the documentation part. Assign me.

Okay @Ripan-Roy, go ahead. I am assigning this issue to the two of you. All the best!

**Comment by Ripan-Roy** [comment]
Sure

**Comment by prajwal-144** [comment]
If possible can I be assigned this issue?
I can work on the temperature detection part.

**Comment by diffrxction** [comment]
> If possible can I be assigned this issue? I can work on the temperature detection part.

Yes, @prajwal-144 you can. Could you possibly give a super simple and rudimentary explanation along with implementation of such a process. Reason being that such a contribution would be more inviting to novice students looking forward to learn more about image processing and thermal imagery.

Like could you try explaining the code itself to some extent in very simple terms? That could act as a guiding step before people read what @tauquirahmed and @Ripan-Roy have contributed earlier today. Thank you.
